src/Metadata/AmazonMetadataBuilder.php

```diff
@@ -114,12 +114,15 @@ protected function getDefaultMetadata()
     /**
      * Gets the correct content-type.
      *
-     * @param string $filename
+     * @param string $filename path to the file inside the s3 bucket
      *
      * @return array
```

Comment:
Bonuspoints if you can add more specific (phpstan) phpdoc here.

Reply:
I would love those bonus points, but what exactly do you mean? Replacing `@return array` with something more specific? Details about why $filename isn't present when this function is called?

Reply:
I think he wants something like
```
@phpstan-return array<string, string>
```
```
@phpstan-return array{'contentType': string}
```

Reply:
You don't need the quotes around `contentType`, but it shoud work